### PR TITLE
LPS-134211 Add Extension to FileEntry Selection

### PR DESCRIPTION
**Pull request by kevenleone**

### CODE REVIEW

**Comment by kevenleone** [comment]
ci:test:sf

**Comment by kevenleone** [comment]
ci:test:relevant

**Comment by kevenleone** [comment]
ci:test:sf

**Comment by kevenleone** [comment]
ci:test:relevant

**Comment by kevenleone** [comment]
ci:stop

**Comment by kevenleone** [comment]
ci:test:sf

**Comment by kevenleone** [comment]
ci:test:relevant

File: modules/apps/digital-signature/digital-signature-web/src/main/java/com/liferay/digital/signature/web/internal/portlet/CollectDigitalSignaturePortlet.java [hunk 1]
```diff
@@ -111,6 +119,25 @@ private String _getTitle(String fileEntryTitle, int count) {
 			new String[] {fileEntryTitle, String.valueOf(count)}, false);
 	}
 
+	private JSONArray _jsonArray(long[] fileEntryIds) throws Exception {
```
Comment (joseabelenda) [inline modules/apps/digital-signature/digital-signature-web/src/main/java/com/liferay/digital/signature/web/internal/portlet/CollectDigitalSignaturePortlet.java:122]: Should be better: `private JSONArray _getJSONArray(long[] fileEntryIds) throws Exception {...`

**Review by joseabelenda — CHANGES_REQUESTED** [review]
@kevenleone 
Great job!
Please, see my comment.

**Comment by kevenleone** [comment]
ci:stop

**Comment by kevenleone** [comment]
ci:test:sf

**Comment by kevenleone** [comment]
ci:forward

**Comment by kevenleone** [comment]
ci:stop

**Pull request closed without merging**